src/components/modal/index.js

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState } from 'react';
+import useTranslation from 'hooks/useTranslation';
+import { z } from 'zod';
+import { useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+
+import Button from 'components/common/Button';
+import smileIcon from 'assets/smilies.svg';
+import Input from 'components/form/Input';
+import TextArea from 'components/form/textarea';
+import { useQuestionMutation } from 'services/questions/questions';
+import close from 'assets/close.svg';
+
+import './styles.scss';
+
+const Modal = ({ handleCloseModal = null }) => {
+  const [showForm, setShowForm] = useState(true);
+  const [sendError, setSendError] = useState(false);
+  const [sendSuccess, setSendSuccess] = useState(false);
```

Comment:
Is this needed? Couldn't you use `isSuccess` and `isError` from `useQuestionMutation`?